udata/core/dataset/models.py

```diff
@@ -568,7 +567,10 @@ class Dataset(Auditable, WithMetrics, DatasetBadgeMixin, Owned, Linkable, db.Doc
     resources = field(db.ListField(db.EmbeddedDocumentField(Resource)), auditable=False)
 
     private = field(db.BooleanField(default=False))
-    frequency = field(db.StringField(choices=list(UPDATE_FREQUENCIES.keys())))
+
+    # FIXME: Do we want required=True to avoid None?
```

Comment:
@maudetes WDYT?

I have to check but I think None is already converted to UNKNOWN via a roundtrip to mongo: StrEnumField.to_python(None) [will return UNKNOWN](https://github.com/opendatateam/udata/pull/3414/files#diff-2d26a6c3d477470033d7130169ec25f9d0590b523d2b664ca73a49cfc6cbaa61R106), so the next to_mongo will end up replacing None with UNKNOWN.

Simple solution IMO is required=True + default=UNKNOWN, but if we want to leave it optional I'll have to fix the current behavior.